webcurator-docs/guides/system-administrator-guide.rst

```diff
@@ -241,241 +215,198 @@ data for the new QA module for each agency, and should be run* **once all
+   The bootstrap user script creates a User with a name of 'bootstrap' and
+   a password of 'password'. Use this account to login to the application
+   once it is up and running. You can use the bootstrap account to create
+   other users and agencies. Once you have setup valid users, it is best to
+   disable the bootstrap user for security reasons.
 
 
-JMX setup
-=========
+Setting up the WCT Application
+==============================
 
-WCT core and every Harvest Agent require JMX Remote access. This means that
-JMX Remote control and access files will need to be setup for the JVM. This is
-done with the following steps:
-
-#.  Create a `jmxremote.password` file by copying the file
-    `jmxremote.password.template` to the jmx remote password file that your
-    installation will use. This template file will be in your JDK's
-    `jre\lib\management` directory.
+Downloading WCT
+---------------
+The binaries for the WCT components can be downloaded from the
+`releases page in the Github repository <https://github.com/WebCuratorTool/webcurator/releases>`_.
 
-    *You can use the property
-    `-Dcom.sun.management.jmxremote.password.file=<property-file>` to point to a
-    different location.*
+Building WCT
+------------
+Alternatively, WCT can be built from source.
 
-    The monitor role and control role have passwords associated with them. These
-    are setting withing hte jmx remote password file::
+To build WCT:
 
-        monitorRole  apassword
-        controlRole  apassword
+- Make sure you have installed and configured Java 1.8 JDK, Maven 3+, and Git.
 
-#.  It is important that this file is protected. If using Windows, refer to the
-    following link to protect the file using the O/S:
-    http://java.sun.com/j2se/1.5.0/docs/guide/management/security-windows.html
+- Clone the code repository from Github using Git::
 
-    If using \*nix platform, protect the file using::
+   git clone https://github.com/WebCuratorTool/webcurator.git
 
-        chmod 600 jmxremote.password.
+- Navigate to the *webcurator-legacy-lib-dependencies/* sub-directory, and run either of
+  the following scripts (depending on your operating system) to install the required
+  legacy dependencies::
 
-#.  Enable the JMX Remote port used in the JVM's startup. Any high port can be
-    used as long as it is unique on the machine that is running the component.
-    The example here uses port `9004`, but if multiple components are running
-    on the same machine, then each component will need a different and unique
-    port number.
+   - install_maven_dependencies.bat
+   - install_maven_dependencies.sh
 
-    For Tomcat, this is done by adding the following to your
-    `$TOMCAT_HOME/bin/catalina.sh script`::
+- Navigate back to the root *webcurator* directory, and build the project using Gradle::
 
-        JAVA_OPTS=-Dcom.sun.management.jmxremote.port=9004
+   gradle clean bootWar --project-prop include<database-type>=true
```

Comment:
It's no longer necessary to include database type when building: all JDBC drivers are included in the binary.

The bootWar target doesn't build everything, currently. In one of the other pending PRs (https://github.com/WebCuratorTool/webcurator/pull/6) I've changed the top-level build command to "gradle clean install".

Reply:
I've updated to reflect this